Python/ast.c

```diff
@@ -2520,6 +2520,8 @@ alias_for_import_name(struct compiling *c, const node *n, int store)
             break;
         case STAR:
             str = PyString_InternFromString("*");
+            if (!str)
```

Comment:
In which case the `if` sentence could fail?

Reply:
In an unlikely case of one-character-string cache miss combined with [memory allocation failure](https://github.com/izbyshev/cpython/blob/830daae1c82ed33deef0086b7b6323e5be0b0cc8/Objects/stringobject.c#L88).